torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -87,10 +91,25 @@ class Raft_Small_Weights(WeightsEnum):
             "recipe": "https://github.com/princeton-vl/RAFT",
             "sintel_train_cleanpass_epe": 2.1231,
             "sintel_train_finalpass_epe": 3.2790,
+            "kitti_train_per_image_epe": 7.6557,
+            "kitti_train_f1-all": 25.2801,
+        },
+    )
+    C_T_V2 = Weights(
+        # Chairs + Things
+        url="https://download.pytorch.org/models/raft_small_C_T_V2-01064c6d.pth",
```

Comment:
Typo?
```suggestion
        url="https://github.com/pytorch/vision/tree/main/references/optical_flow",
```

Reply:
Stupid me, I commented on the wrong line. The remark was for the "recipe" URL which is couple of lines below. I spoke with Nicolas offline and we will fix this on a follow up PR.